ee/modules/650-runtime-audit-engine/images/rules-loader/werf.inc.yaml

```diff
@@ -64,6 +78,6 @@ git:
     - '**/*'
 shell:
   install:
-  - export SOURCE_REPO={{ .SOURCE_REPO }}
-  - git clone --depth 1 {{ $.SOURCE_REPO }}/python-modules/wheels /wheels
-  - pip3 install -f file:///wheels --no-index -r /requirements.txt
+  - apt-get update
+  - apt-get install -y python3 python3-module-pip-run
```

Comment:
Same as this https://github.com/deckhouse/deckhouse/pull/9189/files#r1712683353

Reply:
yes it is.